### PR TITLE
update tikv-server-config file with bottommost-level-compression

### PR DESCRIPTION
**Pull request by ZhenhanGong**
Signed-off-by: zhenhan gong <zhenhan.gong@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

update tikv-server config options.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/tikv/tikv/pull/8908
- Other reference link(s): https://github.com/tikv/tikv/issues/8721

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -888,6 +888,13 @@ bloom filter 为每个 key 预留的长度。
 + 每一层默认压缩算法，默认：前两层为 No，后面 5 层为 lz4。
 + 默认值：["no", "no", "lz4", "lz4", "lz4", "zstd", "zstd"]
 
+### `bottommost-level-compression`
+
++ 设置最后一层的压缩算法，默认：zstd
```
Comment (Connor1996) [inline tikv-configuration-file.md:893]:
```suggestion
+ 设置最后一层的压缩算法
```

Comment (Connor1996) [reply tikv-configuration-file.md:893]: 要说明下为什么要单独设置，compression-per-level 不行么？（这里解释下 dynamic_level_bytes 的影响)

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -888,6 +888,13 @@ bloom filter 为每个 key 预留的长度。
...
++ 会覆盖compression-per-level的设置
```
Comment (Connor1996) [inline tikv-configuration-file.md:894]:
```suggestion
+ 会覆盖 compression-per-level 的设置
```
中英文间要空格

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -888,6 +888,13 @@ bloom filter 为每个 key 预留的长度。
...
++ 如果不想设置最后一层的压缩算法，使用"disable"
```
Comment (Connor1996) [inline tikv-configuration-file.md:895]:
```suggestion
+ 如果不想设置最后一层的压缩算法，使用 "no"
```
跟前面 compression-per-level 保持一致吧

Comment (yiwu-arbug) [reply tikv-configuration-file.md:895]: 这里含义不一样的，"no” 是说不压缩，"disable" 是说最后一层压缩不特殊处理。

Comment (Connor1996) [reply tikv-configuration-file.md:895]: 喔喔

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -888,6 +888,24 @@ bloom filter 为每个 key 预留的长度。
 + 每一层默认压缩算法，默认：前两层为 No，后面 5 层为 lz4。
 + 默认值：["no", "no", "lz4", "lz4", "lz4", "zstd", "zstd"]
 
+### `bottommost-level-compression`
+
++ 设置最后一层的压缩算法，会覆盖 compression-per-level 的设置
```
Comment (yiwu-arbug) [inline tikv-configuration-file.md:893]: 感觉解释得有点太 detail 太啰嗦，有可能简略一点吗

Comment (TomShawn) [reply tikv-configuration-file.md:893]: 赞同 @yiwu-arbug 。
@ZhenhanGong 能否只提供影响用户行为的信息呢？请尽量避免（从工程师角度）在用户文档中提供过多实现原理的信息。

Comment (ZhenhanGong) [reply tikv-configuration-file.md]: 嗯嗯好的，简化了

**Review by Connor1996 — APPROVED** [review]

**Review by yiwu-arbug — COMMENTED** [review]
LGTM

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -888,6 +888,15 @@ bloom filter 为每个 key 预留的长度。
 + 每一层默认压缩算法，默认：前两层为 No，后面 5 层为 lz4。
 + 默认值：["no", "no", "lz4", "lz4", "lz4", "zstd", "zstd"]
 
+### `bottommost-level-compression`
+
++ 设置最下一层的压缩算法，会覆盖 compression-per-level 的设置
++ 考虑到 level-compression-dynamic-level-bytes 为 true 时，最下层
```
Comment (yiwu-arbug) [inline tikv-configuration-file.md:894]: typo
```suggestion
+ 考虑到 level-compaction-dynamic-level-bytes 为 true 时，最下层
```

Comment (ZhenhanGong) [reply tikv-configuration-file.md:894]: My bad, I will fix that.

File: tikv-configuration-file.md [hunk 6]
```diff
@@ -888,6 +888,15 @@ bloom filter 为每个 key 预留的长度。
 + 每一层默认压缩算法，默认：前两层为 No，后面 5 层为 lz4。
 + 默认值：["no", "no", "lz4", "lz4", "lz4", "zstd", "zstd"]
 
+### `bottommost-level-compression`
+
++ 设置最下一层的压缩算法，会覆盖 compression-per-level 的设置
++ 考虑到当 level-compaction-dynamic-level-bytes 为 true 时，最下层
```
Comment (TomShawn) [inline tikv-configuration-file.md:894]: 本文档没有 `level-compaction-dynamic-level-bytes` 这个配置项，是否需要一起添加进来呢？ @yiwu-arbug @Connor1996

File: tikv-configuration-file.md [hunk 7]
```diff
@@ -888,6 +888,15 @@ bloom filter 为每个 key 预留的长度。
...
++ 并不是一开始就直接采用 compression-per-level[6] 的压缩算法，所以提供
```
Comment (TomShawn) [inline tikv-configuration-file.md:895]: `[6]` 是什么意思呢？

Comment (ZhenhanGong) [reply tikv-configuration-file.md:895]: 这是个数组，数组第七个元素

File: tikv-configuration-file.md (hunk 6 above)
Comment (yiwu-arbug) [reply tikv-configuration-file.md:894]: 那就不要提这个选项，当作默认值是true的情况讨论。我们不希望用户修改这个配置。

File: tikv-configuration-file.md (hunk 7 above)
Comment (TomShawn) [reply tikv-configuration-file.md:895]: 能否写成更贴近用户的语言呢？

Comment (ZhenhanGong) [reply tikv-configuration-file.md]: > 能否写成更贴近用户的语言呢？比如：“并不是一开始就直接采用 `compression-per-level` 指定的第七层压缩算法”？

嗯嗯，修改了

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**